src/commonMain/sqldelight/com/wizbii/cinematic/journey/data/migrations/3.sqm

```diff
@@ -0,0 +1 @@
+ALTER TABLE tmdbMovieRecord ADD COLUMN budget INTEGER;
```

Comment:
Version 3 already exists, version 4 should be used instead

Reply:
Updated